api/data_refinery_api/views.py

```diff
@@ -192,7 +194,13 @@ def get(self, request, format=None):
         filter_dict = request.query_params.dict()
         filter_dict.pop('limit', None)
         filter_dict.pop('offset', None)
-        samples = Sample.objects.filter(**filter_dict)
+        ids = self.request.query_params.get('ids', None)
+
+        if ids is not None:
+            ids = [ int(x) for x in ids.split(',')]
+            samples = Sample.objects.filter(pk__in=ids)
+        else:
+            samples = Sample.objects.filter(**filter_dict)
```

Comment:
Ideally, you should be able to handle both types of filter at the same time here, rather than just one or the other..